muellermartin/nosystemd.org
# Add Funtoo Linux Distro and Removing Project Trident

Summary
========
Features included in this change:
* Fixes #43 
  * Add Funtoo Linux as an alternative nosystemd Linux distribution
* Remove dormant/dead Project Trident from alternative Linux Distrio list as domain is offline

Test Plan
=======
* Verify the new `index.html` is rendering properly with `xdg-open htdocs/index.html`
![shot-2022-04-12_01-02-17](https://user-images.githubusercontent.com/700847/162912456-7cf2d9be-fc37-4213-ac3e-8c7b7449b0f9.jpg)
* Run `./checklinks.py` to verify all `index.html` links:
```
Checking URL https://www.funtoo.org/
OK
Checking URL https://project-trident.org/
Can't connect to https://project-trident.org/
FAIL
```
